modules/sbo-release-notes-1-3.adoc

```diff
@@ -0,0 +1,12 @@
+[id="sbo-release-notes-1-3_{context}"]
+// Module included in the following assembly:
+//
+// * applications/connecting_applications_to_services/sbo-release-notes.adoc
+:_content-type: REFERENCE
+= Release notes for {servicebinding-title} 1.3
+
+{servicebinding-title} 1.3 is now available on {product-title} 4.9, 4.10, and 4.11.
+
+[id="deprecation-notice-1-3_{context}"]
+== Deprecated functionality
+* In {servicebinding-title} 1.3, the Operator Lifecycle Manager (OLM) descriptor feature is deprecated and is planned to be removed in a future release. Red Hat will provide bug fixes and support for this feature during the current release lifecycle, but this feature will no longer receive enhancements and will be removed. As an alternative to OLM descriptors, you can use CRD annotations instead to declare binding data.
```

Comment:
We are not deprecating, we are removing. Please use the text in release notes here https://issues.redhat.com/browse/APPSVC-1203. This feature is not being used today

Reply:
@dperaza4dustbit, we can not use the same RN text as mentioned in https://issues.redhat.com/browse/APPSVC-1203. I have modified the RN statement as per our supplemetary style guide. Please have a look.